tree/treeplayer/src/TTreeFormula.cxx

```diff
@@ -2892,7 +2905,8 @@ Int_t TTreeFormula::DefinedVariable(TString &name, Int_t &action)
                return -3;
             }
 
-            fManager->Add(subform);
+            if (fManager)
+               fManager->Add(subform);
```

Comment:
```suggestion
            fManager->Add(subform);
```
`fManager` is set in the constructor and assume to be not a `nullptr` throughout the code.

Reply:
It crashed for me during the ctest otherwise?

Reply:
With the default constructor